src/proxy/inbound.rs

```diff
@@ -372,14 +372,14 @@ impl Inbound {
                 }
                 // This check should be removed in favor of an L4 policy check
                 // We should express as policy whether or not traffic is allowed to bypass a waypoint
-                if has_waypoint && !from_waypoint {
-                    info!(%rbac_ctx.conn, "bypassed waypoint");
-                    connection_manager.release(&rbac_ctx).await;
-                    return Ok(Response::builder()
-                        .status(StatusCode::UNAUTHORIZED)
-                        .body(Empty::new())
-                        .unwrap());
-                }
+                // if has_waypoint && !from_waypoint {
+                //     info!(%rbac_ctx.conn, "bypassed waypoint");
+                //     connection_manager.release(&rbac_ctx).await;
+                //     return Ok(Response::builder()
+                //         .status(StatusCode::UNAUTHORIZED)
+                //         .body(Empty::new())
+                //         .unwrap());
+                // }
```

Comment:
What do we think about this? Something similar will be necessary at some point but are we comfortable doing that before istiod is able to generate a L4 policy with similar effects?

cc @louiscryan @howardjohn @stevenctl

Reply:
If we drop this, we should drop `debug!("request from waypoint, skipping policy")`

Reply:
yeah, that tracks

Reply:
I had been testing with this but forgot it wasn't pushed yet. Pushed now.